.github/2.0/cookbooks/Daemon.md

```diff
@@ -2,60 +2,159 @@
 <!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
 Table of Contents
 
-- [JinaD 2.0](#jinad-20)
+- [JinaD (Daemon)](#jinad-daemon)
+  - [Using JinaD](#using-jinad)
+    - [Run](#run)
+    - [API Docs](#api-docs)
+    - [JinaDClient](#jinadclient)
+      - [Connect from local](#connect-from-local)
+      - [Get remote status](#get-remote-status)
   - [Concepts](#concepts)
     - [Workspace](#workspace)
+      - [Special files](#special-files)
+      - [Create a workspace (redoc)](#create-a-workspace-redoc)
+      - [Get the details of a workspace (redoc)](#get-the-details-of-a-workspace-redoc)
+      - [List all workspaces (redoc)](#list-all-workspaces-redoc)
+      - [Delete a workspace (redoc)](#delete-a-workspace-redoc)
     - [RESTful Flows](#restful-flows)
-    - [RESTful Pods/Peas](#restful-podspeas)
-  - [Using JinaD](#using-jinad)
-    - [Run](#run)
-    - [Example Usage](#example-usage)
+      - [Create a Flow (redoc)](#create-a-flow-redoc)
+      - [Get details of a Flow (redoc)](#get-details-of-a-flow-redoc)
+      - [Terminate a Flow (redoc)](#terminate-a-flow-redoc)
+    - [RESTful Pods/Peas (Advanced)](#restful-podspeas-advanced)
+      - [Get all accepted arguments (redoc)](#get-all-accepted-arguments-redoc)
+      - [Create a Pea/Pod (redoc)](#create-a-peapod-redoc)
+      - [Get details of a Pea/Pod (redoc)](#get-details-of-a-peapod-redoc)
+      - [Terminate a Pea/Pod (redoc)](#terminate-a-peapod-redoc)
+    - [Logstreaming](#logstreaming)
     - [Development using JinaD](#development-using-jinad)
+        - [Build](#build)
+        - [Run](#run-1)
+        - [Why?](#why)
 
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
-# JinaD 2.0
+# JinaD (Daemon)
 
 JinaD is a daemon for deploying & managing Jina on remote via a RESTful interface. It achieves isolation of deployments by defining a `workspace` for each Jina object.
 
 ------
 
+## Using JinaD
+
+`JinaD` docker image is published on [Docker Hub](https://hub.docker.com/r/jinaai/jina/tags?page=1&ordering=last_updated&name=-daemon) along with core & follows the [standard image versioning](https://github.com/jina-ai/jina/blob/master/RELEASE.md#docker-image-versioning) used in Jina.
+
+### Run
+
+To deploy JinaD, SSH into a server (e.g.- ec2 instance) & run the below command.
```

Comment:
Not sure, but maybe we should point also here to the options to change the port and root workspace directory? People might have reasons to use or not use 8000 and /tmp/jinad and will wonder this when reading this part. (maybe just link somewhere with the args or so)